change/@fluentui-react-shared-contexts-4e124152-f0bd-4310-9a56-2ce177884ec7.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 3 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27581&commitId=66ea2cc68c329614864f6b1e47c71e8ef8298827&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1930 screenshots unchanged, 0 screenshots with different dimensions and 3 screenshots with visible difference.


  <details><summary><b>unknown</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|AvatarGroup Converged.overflowContent - Dark Mode.popoverContentOpen.chromium.png|1825| Changed|
|AvatarGroup Converged.overflowContent - High Contrast.popoverContentOpen.chromium.png|2820| Changed|
|AvatarGroup Converged.overflowContent.popoverContentOpen.chromium.png|1985| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>